src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -655,23 +655,23 @@ m_lastInputTime  (0)
         Atom WMHintsAtom = getAtom("_MOTIF_WM_HINTS", false);
         if (WMHintsAtom)
         {
-            static const unsigned long MWM_HINTS_FUNCTIONS   = 1 << 0;
-            static const unsigned long MWM_HINTS_DECORATIONS = 1 << 1;
-
-            //static const unsigned long MWM_DECOR_ALL         = 1 << 0;
-            static const unsigned long MWM_DECOR_BORDER      = 1 << 1;
-            static const unsigned long MWM_DECOR_RESIZEH     = 1 << 2;
-            static const unsigned long MWM_DECOR_TITLE       = 1 << 3;
-            static const unsigned long MWM_DECOR_MENU        = 1 << 4;
-            static const unsigned long MWM_DECOR_MINIMIZE    = 1 << 5;
-            static const unsigned long MWM_DECOR_MAXIMIZE    = 1 << 6;
-
-            //static const unsigned long MWM_FUNC_ALL          = 1 << 0;
-            static const unsigned long MWM_FUNC_RESIZE       = 1 << 1;
-            static const unsigned long MWM_FUNC_MOVE         = 1 << 2;
-            static const unsigned long MWM_FUNC_MINIMIZE     = 1 << 3;
-            static const unsigned long MWM_FUNC_MAXIMIZE     = 1 << 4;
-            static const unsigned long MWM_FUNC_CLOSE        = 1 << 5;
+            static constexpr unsigned long MWM_HINTS_FUNCTIONS   = 1 << 0;
+            static constexpr unsigned long MWM_HINTS_DECORATIONS = 1 << 1;
+
+            //static constexpr unsigned long MWM_DECOR_ALL         = 1 << 0;
+            static constexpr unsigned long MWM_DECOR_BORDER      = 1 << 1;
+            static constexpr unsigned long MWM_DECOR_RESIZEH     = 1 << 2;
+            static constexpr unsigned long MWM_DECOR_TITLE       = 1 << 3;
+            static constexpr unsigned long MWM_DECOR_MENU        = 1 << 4;
+            static constexpr unsigned long MWM_DECOR_MINIMIZE    = 1 << 5;
+            static constexpr unsigned long MWM_DECOR_MAXIMIZE    = 1 << 6;
+
+            //static constexpr unsigned long MWM_FUNC_ALL          = 1 << 0;
+            static constexpr unsigned long MWM_FUNC_RESIZE       = 1 << 1;
+            static constexpr unsigned long MWM_FUNC_MOVE         = 1 << 2;
+            static constexpr unsigned long MWM_FUNC_MINIMIZE     = 1 << 3;
+            static constexpr unsigned long MWM_FUNC_MAXIMIZE     = 1 << 4;
+            static constexpr unsigned long MWM_FUNC_CLOSE        = 1 << 5;
```

Comment:
I guess we can do this here, because the Unix impl won't be compiled by VS 2019?

Still a bit strange to use different "styles" across the code base, this may not be obvious to someone who wasn't involved in the discussions we have now 🧐

Reply:
Nah, the problem with having to add `inline` was specific to out-of-class `constexpr` definitions of non-`constexpr` static data members, these are in function-scope, so it's all good.